debian/postinst

```diff
@@ -16,7 +17,20 @@ gunzip <"$base_img" | cpio -idm
 
 mknod -m 600 ./dev/console c 5 1
 
-modpath="/lib/modules/$(uname -r)/"
+. /usr/lib/grub/grub-mkconfig_lib
+
+#
+# Determine the latest installed kernel version, and sync kernel modules for
+# that version into the recovery environment.
+#
+list=
+for i in /boot/vmlinuz-* /vmlinuz-* /boot/kernel-*; do
```

Comment:
We should look exclusively for `/boot/vmlinuz-*` here, given that this is where we are installing the file. If for some reason we get hits in other directories as well, `version_find_latest` will likely not work properly, and if we get a hit for `/boot/kernel-*` this line will break:
``` modpath="/lib/modules/${basename#vmlinuz-}" ```